community/kernel-api/src/test/java/org/neo4j/internal/kernel/api/RelationshipTestSupport.java

```diff
@@ -25,7 +25,11 @@
 import java.util.Map;
 import java.util.function.Function;
 
-import org.neo4j.graphdb.*;
+import org.neo4j.graphdb.Direction;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.graphdb.Node;
```

Comment:
Time to left IntelliJ know about this rule?

Reply:
Strange thing is I never got that before, and fixed it with `optimize imports`, so I think IntelliJ knows. Dunno if I somehow managed to switch to default code style for a period when that happened...